src/diffusers/schedulers/scheduling_heun_discrete.py

```diff
@@ -22,8 +23,11 @@
 from .scheduling_utils import KarrasDiffusionSchedulers, SchedulerMixin, SchedulerOutput
 
 
-# Copied from diffusers.schedulers.scheduling_ddpm.betas_for_alpha_bar
```

Comment:
Let's not remove the ` # Copied from ` here but instead change it in `scheduling_ddpm` right away